templates/kube-vip/kube-vip.yml.j2

```diff
@@ -72,6 +72,9 @@ spec:
 {% endif %}
         image: "{{ rke2_kubevip_image }}"
         name: kube-vip
+        ports:
+        - name: metrics
+          containerPort: 2112
```

Comment:
I haven't found it in the kube-vip documentation but the port seems to be configurable. https://github.com/kube-vip/kube-vip/blob/a5ec16e1e835d88a57bf812862ade8420bd17759/cmd/kube-vip.go#L150

Please make the port as a variable with `2112` value as a default

Reply:
Note: this would also need to be changed in L63 due to kube-vip running on `hostNetwork: true`

Reply:
Make sense